tests/PHPStan/Rules/Functions/data/bug-10298.php

```diff
@@ -0,0 +1,18 @@
+<?php declare(strict_types = 1);
+
+namespace Bug10298;
+
+#[\Attribute(\Attribute::TARGET_PROPERTY)]
+class PropAttr {}
+
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class ParamAttr {}
+
+class Test
+{
+	public function __construct(
+		#[PropAttr]
+		#[ParamAttr]
+		public string $test
```

Comment:
Coming from this issue [phpstan/phpstan#10385](https://github.com/phpstan/phpstan/issues/10385) maybe you could improve the test suite by adding another test case to cover when an attribute having both property and parameter targets like:

```php
#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_PARAMETER)]
class PropAndParamAttr {}

class Test
{
	public function __construct(
		#[PropAndParamAttr]
		public string $test
	) {}
}
```

Reply:
I believe there is already such a test. Feel free to check that and send a PR in case it isn't 😊